### ISSUE TITLE
How do I tag people in my photo?

### ISSUE THREAD

**Issue by sekon-am** [mgp25/Instagram-API#74]
I'm sorry, but I can't find this function. Is it done?

**Comment by mgp25** [comment]
I dont think ill ever add that

Regards

**Comment by sekon-am** [comment]
why? it's extremely useful

**Comment by dfanara** [comment]
In all fairness if you think it would be extremely useful you could implement it yourself and always submit a pull request.

**Comment by sekon-am** [comment]
I can do it if you'll provide me request details

**Comment by dfanara** [comment]
Maybe since @mgp25 is familiar with reverse engineering the Instagram app he will be willing to provide the endpoints.

**Comment by sekon-am** [comment]
Would be great

**Comment by alexleosumy** [comment]
I'm looking for a function to tag people too. Would be great if you will add it

**Comment by mgp25** [comment]
The problem is the following: i did this API just for upload photos to my account, i dont pretend to do a full API although i did most of it, i posted most of the information in this repository, if you want any endpoint is just capturing the traffic using a proxy.

I'll accept all PR to complete the API. It's quite easy :yum:

**Comment by sekon-am** [comment]
> if you want any endpoint is just capturing the traffic using a proxy
I've tried to sniff an android Instagram application use web proxy.
Unfortunately, it uses TLS, so I can't sniff any endpoints. If you know how to sniff applications that used TLS than your explanations are welcome!
So I decided that Android app reversing is a true way to get endpoints, which one you use (I hope so). So would be great to get tag people endpoint. I'll add it to the  code myself.
